pkg/services/alerting/notifiers/hipchat.go

```diff
@@ -144,6 +162,7 @@ func (this *HipChatNotifier) Notify(evalContext *alerting.EvalContext) error {
 
 	hipUrl := fmt.Sprintf("%s/v2/room/%s/notification?auth_token=%s", this.Url, this.RoomId, this.ApiKey)
 	data, _ := json.Marshal(&body)
+	this.log.Debug(string(data))
```

Comment:
Its preferred to describe what your logging.
`this.log.Info("Request payload", "json", string(data))`

Reply:
done